subxt/src/client/online_client.rs

```diff
@@ -3,9 +3,10 @@
 // see LICENSE for license details.
 
 use super::{OfflineClient, OfflineClientT};
+use crate::backend::StreamOfResults;
```

Comment:
nit: I think we can move these imports back to `crate::backend` below. I do like the grouping of rustfmt nightly for these